src/azul/service/manifest_controller.py

```diff
@@ -137,28 +140,41 @@ def get_manifest_async(self,
 
         if manifest is None:
             location = furl(self_url, args={'token': token.encode()})
+            unix_cmd = f'curl "{location}"'
+            windows_cmd = f'curl.exe "{location}"'
             body = {
                 'Status': 301,
                 'Location': location.url,
                 'Retry-After': token.wait_time
             }
         else:
-            if fetch:
+            try:
+                query_params['objectKey']
+            except KeyError:
                 url = self.manifest_url_func(fetch=False,
                                              catalog=manifest.catalog,
                                              format_=manifest.format_,
                                              filters=json.dumps(manifest.filters),
                                              objectKey=manifest.object_key)
             else:
                 url = manifest.location
+            command_lines = self.service.command_lines(manifest, url)
+            unix_cmd = command_lines['bash']
+            windows_cmd = command_lines['cmd.exe']
```

Comment:
A bit annoying that we have to unpack like this, but I can't think of a simple way to do it that's any cleaner.